tiles/src/main/java/com/protomaps/basemap/feature/QrankDb.java

```diff
@@ -26,9 +26,16 @@ public long get(long wikidataId) {
     return this.db.get(wikidataId);
   }
 
-  public long get(String wikidataId) {
-    long id = Long.parseLong(wikidataId.substring(1));
-    return this.get(id);
+  public long get(String osmValue) {
+    try {
+      if (osmValue.contains(";")) {
+        osmValue = osmValue.split(";")[0];
+      }
+      long id = Long.parseLong(osmValue.substring(1));
```

Comment:
Wikidata IDs usually start with a `Q`? In the linked issue the 0 index position values lacks the Q, while the 1st index position has it. Might need special logic around this (eg to add or strip Q prefix and treat like an int?)

(repeating here in PR instead of earlier comment on a commit)

Reply:
Version 5 here is the actual OSM entity: https://www.openstreetmap.org/way/461759932/history which has Qs in both places